librz/util/buf.c

```diff
@@ -541,7 +541,6 @@ static st64 buf_format(RzBuffer *dst, RzBuffer *src, const char *fmt, int n) {
 			}
 
 			m = 1;
-			bigendian = true;
```

Comment:
is this really dead? I'm not sure 100%

Reply:
bigendian is set in the switch line 461

Reply:
it is not always set. Some cases do not set it and it may be used after that AFAIS.

Reply:
case '0' - '9' -> continue (loop)
case 's', 'S', 'i', 'I', 'l', 'L', 'c' -> set bigendian
default -> return from the function

Reply:
There is no side effect if the line 544 is removed

Reply:
What happens if there is a number and then the fmt string ends? In that case, the previously set bigendian value would be considered, no?

Reply:
bigendian is a local loop variable (line 458)

So the case of a number and then fmt strings ends as no impact

Reply:
I don't see what I am missing.

Reply:
Ok I think you are right!